frontend/components/App/FishbowlPreJoin/form-auth.tsx

```diff
@@ -16,15 +17,15 @@ import { InputStyled } from 'ui/Form';
 import Button from 'ui/Button';
 import FormikForm from 'ui/Form';
 
-type TProps = {
+interface Props {
   name: string;
 };
 
-const AuthUser = ({ name }: TProps) => {
+const AuthUser: React.FC<Props> = ({ name }) => {
```

Comment:
What do you think about using AuthUserForm and AuthUserProps?

Reply:
Normally, while props are only in the component scope, they aren't needed to be specified like `AuthUserProps` .

There is another case of some forms/components where we need more than one interface per file then yes I'd do it this way.

I agree with AuthUserForm name.